_episodes/01-intro.md

```diff
@@ -89,14 +89,14 @@ when typing commands, either from these lessons or from other sources,
 The part that you type,
 `ls -F /` in the second line of the example,
 typically has the following structure: a **command**,
-some **flags** (also called **options** or **switches**) and an **argument**.
+some **options** (also called **switches** or **flags**) and an **argument**.
 Flags start with a single dash (`-`) or two dashes (`--`), and change the behaviour of a command.
 Arguments tell the command what to operate on (e.g. files and directories).
-Sometimes flags and arguments are referred to as parameters.
-A command can be called with more than one flag and more than one argument: but a
-command doesn't always require an argument or a flag.
+Sometimes options and arguments are referred to as parameters.
```

Comment:
Maybe:
Sometimes options and arguments are together referred to as **parameters**.